README.rst

```diff
@@ -317,6 +273,7 @@ third-party libraries may not have the same license as vineyard.
 .. _Installation: https://github.com/v6d-io/v6d/blob/main/docs/notes/install.rst
 .. _Contributing: https://github.com/v6d-io/v6d/blob/main/CONTRIBUTING.rst
 .. _Getting Started: https://v6d.io/notes/getting-started.html
+.. _Vineyard Operator: https://github.com/dashanji/v6d/blob/add-vineyard-operator-doc/docs/notes/vineyard-operator.rst
```

Comment:
Don't link to your own fork.

Reply:
Yeah, The link is only for testing CI, I think the next commit will fail as the link of vineyard is deadline.